app/controllers/api/v1/brands_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Api::V1::BrandsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
```

Comment:
this will most likely be used by _all_ of your API controllers meaning it can be added to the inherited `ApplicationController`!